hypervisor/virt/hyperv/hypervcli.py

```diff
@@ -0,0 +1,104 @@
+import json
+import re
+
+from hypervisor import logger
+from hypervisor.ssh import SSHConnect
+
+
+class HypervCLI:
+    def __init__(self, server, ssh_user, ssh_passwd):
+        """
+        Collect hyperv information, provide Guest add/delete/start/stop/suspend/resume functions
+        via PowerCli Command line
+        :param server: the ip for the hyperv server
+        :param ssh_user: the ssh user for the hyperv server
+        :param ssh_passwd: the ssh password for the hyperv server
+        """
+        self.server = server
+        self.ssh_user = ssh_user
+        self.ssh_passwd = ssh_passwd
+        self.ssh = SSHConnect(self.server, user=self.ssh_user, pwd=self.ssh_passwd)
+
+    def _format(self, ret=0, stdout=None):
+        """
+        Convert the json string to python list data
+        :param ret: return code
+        :param stdout: output for the execute command
+        :return: the list after json.loads
+        """
+        stdout = re.findall(r'[[][\W\w]+[]]', stdout)[0]
+        if ret is 0 and stdout is not None:
+            return json.loads(stdout)
+
+    def info(self):
+        """
+        Get the VMhost info
+        :return: the VMHosts info
+        """
+        cmd = 'PowerShell ConvertTo-Json @(Get-VMHost)'
+        ret, output = self.ssh.runcmd(cmd)
+        output = self._format(ret, output)
+        for host in output:
+            logger.info(f"Get Hyperv master: {host['Name']}")
+        return output
+
+    def guest_search(self, guest_name, uuid_info=False):
+        """
+        Search the specific guest, return the expected attributes
+        :param guest_name: name for the specific guest
+        :param uuid_info: if you need the uuid_info: guest_uuid, hyperv_uuid, hyperv_hwuuid
```

Comment:
Is there hwuuid for hyperv, virt-who just support hwuuid for esx and rhevm.

Reply:
nope, will delete it from the docstring